src/main/java/nextstep/optional/ComputerStore.java

```diff
@@ -21,6 +23,10 @@ public static String getVersion(Computer computer) {
     }
 
     public static String getVersionOptional(Computer computer) {
-        return null;
+        return Optional.ofNullable(computer)
+                .map(Computer::getSoundcard)
+                .map(Soundcard::getUsb)
+                .map(USB::getVersion)
+                .orElse(UNKNOWN_VERSION);
```

Comment:
`Computer`에서 `getUsbVersion`를 만들어서 아래처럼 써도 좋을 것 같아요. 
```suggestion
        return Optional.ofNullable(computer)
                .map(Computer::getUsbVersion)
                .orElse(UNKNOWN_VERSION);
```